crates/filtering/src/hysteresis.rs

```diff
@@ -1,29 +1,58 @@
-pub fn greater_than_with_hysteresis(
+pub fn greater_than_with_hysteresis_from_tresholds(
+    last_evaluation: bool,
+    value: f32,
+    lower_threshold: f32,
+    upper_threshold: f32,
+) -> bool {
+    if value > upper_threshold {
+        true
+    } else if value < lower_threshold {
+        false
+    } else {
+        last_evaluation
+    }
+}
+
+#[allow(clippy::if_same_then_else)]
+pub fn less_than_with_hysteresis_from_thresholds(
+    last_evaluation: bool,
+    value: f32,
+    lower_threshold: f32,
+    upper_threshold: f32,
+) -> bool {
+    if value < lower_threshold {
+        true
+    } else if value > upper_threshold {
+        true
```

Comment:
are you sure?! Better add some test cases that these mistakes do not happen

Reply:
Which mistakes? 🤔
I just asked my flatmate, he thinks its easy enough that we don't need any test cases...

Reply:
Ah ops copy pasta

Reply:
I still would like to see test cases ;)

Reply:
Done